src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="../src/images/imac.jpeg"
+        alt="apple"
+      />
+      <div class="card__product-description">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
```

Comment:
Still not fixed from the previous review
```suggestion
      <h2 class="card__product-description">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
```